src/prefabs/camera-tool.js

```diff
@@ -1,4 +1,5 @@
 /* eslint-disable react/prop-types */
+/* eslint-disable react/no-unknown-property */
```

Comment:
entity is defined in a separate TS file, however the latest React complains these defined properties are unknown. Disable eslint for now.

Reply:
Yeah for the entity stuff we are using JSX but **not**. I assume we just got some stronger linting rules now that are being applied. It would definitely be nice if we could have linting working for this stuff again at some point. I would even favor modifying the defaults to be more permissive removing some react rules.